cgmanifest.json

```diff
@@ -13618,6 +13618,16 @@
         }
       }
     },
+    {
+      "component": {
+        "type": "other",
+        "other": {
+          "name": "msft-golang",
+          "version": "1.20.11",
+          "downloadUrl": "https://github.com/microsoft/go/releases/download/v1.20.11-1/go.20231107.4.src.tar.gz"
```

Comment:
Shouldn't this replace the 1.19.12 version entry above?

Reply:
No, I left the 1.19.11 version, since that is now being used for the second step in the bootstrap process. (1.4 -> 1.19.12 -> 1.20.11)